### ISSUE TITLE
Please add handler for disabling cron.{allow|deny} logic

### ISSUE THREAD

**Issue by gzhukov** [pegasd/puppet-cron#2]
Hi, cron.{allow|deny} is good, but sometimes we don't want to manage crontabs in such security way (on dev machines for example). Could you add a flag for disabling this logic?

**Comment by pegasd** [comment]
I agree that the current implementation is a bit wacky. I'll try to think of a better way to manage this.

**Comment by pegasd** [comment]
This has been fixed in `0.6.1` release.
I added `$cron::allow_all_users` parameter.

You can use the following class declaration to enable the desired behavior:

```puppet
class { 'cron':
  allow_all_users => true,
  purge_cron      => false, # this is needed so that users' crontabs won't be purged
}
```

or add these to hiera for a specific host:

```yaml
cron::allow_all_users: true
cron::purge_cron: false
```
